client/db/src/utils.rs

```diff
@@ -320,7 +320,9 @@ pub fn require_header<Block: BlockT>(
 	id: BlockId<Block>,
 ) -> sp_blockchain::Result<Block::Header> {
 	read_header(db, col_index, col, id)
-		.and_then(|header| header.ok_or_else(|| sp_blockchain::Error::UnknownBlock(format!("{}", id))))
+		.and_then(|header| header.ok_or_else(||
+			sp_blockchain::Error::UnknownBlock(format!("Require header: {}", id))
```

Comment:
Isn't it supposed to be just block id in this error?
Looks strange, is it `UnknownBlock` or "Require header" error now?

Reply:
It's `UnknownBlock` in the context of `Require header`. We use the same error type in so many places that it's hard to understand which place triggers the error currently. So I wanted to augment the error messages, without affecting the semantics.